### DISCUSSION TITLE
How to unselect codes and remove link for specific line in url?

### DISCUSSION THREAD

**Discussion by kwonmha** [community/community#64905] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![스크린샷 2023-08-25 오전 9 43 28](https://github.com/community/community/assets/8953934/f267bd85-4e17-4911-9c44-1cf26675be9c)

If I select a line in codes, yellow line appears and "#L1" is added to url.
How to unselect codes and remove those markers(yellow line and line info on url)?

**Comment by ggdio** [comment]
Hello, just click "Preview" tab, then return back to "Code" tab in github, that will do the trick !

**Reply by kwonmha** [reply]
TY.
Good idea but that problem usually happens when I read codes where there are no "Preview" tab.
In the screenshot, it's `.md` file which has "Preview" tab.

**Reply by ggdio** [reply]
Yeah, thats sad =(

I tried a few things but anything worked, im pretty sure we will have to accept that issue
